apps/src/aichat/views/ChatEventsList.tsx

```diff
@@ -63,50 +73,34 @@ const ChatEventsList: React.FunctionComponent<ChatEventsListProps> = ({
     );
   };
 
+  const handleParentKeyDown = (e: React.KeyboardEvent<HTMLDivElement>) => {
+    if (e.key === 'Enter') {
```

Comment:
If a user is tabbing through 'Copy' buttons in the chat workspace (not yet in chat navigation mode) and presses 'Enter', they aren't able to copy the message text and focus goes to last message in chat navigation mode. However, they are able to press the spacebar to copy. Is that the expectation?

If not, should we check whether the user is already in chat navigation mode or not when they hit 'Enter' on a copy button?